packages/tradershub/src/utils/cfd.ts

```diff
@@ -0,0 +1,12 @@
+import { PlatformDetails } from '../features/cfd/constants';
+import { TPlatforms } from '../types';
+
+export const platformPasswordResetRedirectLink = (platform: TPlatforms.All, isVirtual?: boolean) => {
+    switch (platform) {
+        case PlatformDetails.mt5.platform:
+            return isVirtual ? 11 : 10;
```

Comment:
These redirections used to be used when we had dashboard for CFDs. These redirections need to be all directed `traders-hub`

Reply:
The redirections are already handled in redirect.jsx in core

Reply:
The redirection is pointed to tradersHub but we need to pass these params as BE need these params. Then it is redirected to TradersHub which is through redirect.jsx.

Correct me if I am wrong?

Reply:
yes